src/modules/launchList.js

```diff
@@ -128,6 +128,9 @@ export default class Launch {
   }
 
   static async getComments() {
+    const getDetails = document.querySelector('.commentList');
+        getDetails.innerHTML = ' ';
+        commentsCount.innerText = '0';
```

Comment:
The implementation here is to clear the comments and set counter to 0 if there are no comments.